libs/dh/dev-examples/feature-processes/src/details/details.component.ts

```diff
@@ -0,0 +1,212 @@
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { afterRenderEffect, Component, computed, inject, input, viewChild } from '@angular/core';
+
+import { TranslocoDirective, TranslocoPipe } from '@ngneat/transloco';
+
+import {
+  WattDescriptionListComponent,
+  WattDescriptionListItemComponent,
+} from '@energinet-datahub/watt/description-list';
+
+import {
+  WattProgressTrackerComponent,
+  WattProgressTrackerStepComponent,
+} from '@energinet-datahub/watt/progress-tracker';
+
+import { WattDatePipe } from '@energinet-datahub/watt/date';
+import { VaterFlexComponent } from '@energinet-datahub/watt/vater';
+import { WATT_DRAWER, WattDrawerComponent } from '@energinet-datahub/watt/drawer';
+
+import { lazyQuery } from '@energinet-datahub/dh/shared/util-apollo';
+import { DhProcessStateBadge } from '@energinet-datahub/dh/wholesale/shared';
+import { DhNavigationService } from '@energinet-datahub/dh/shared/navigation';
+import { GetProcessDocument } from '@energinet-datahub/dh/shared/domain/graphql';
+import { DhEmDashFallbackPipe, DhResultComponent } from '@energinet-datahub/dh/shared/ui-util';
+
+import type {
+  DhProcessCalculation,
+  DhProcessWholesaleRequest,
+  DhProcessEnergyTimeSeriesRequest,
+} from '../types';
+
+import { DhCalculationsDetailsGridAreasComponent } from './gridareas.component';
+
+@Component({
+  imports: [
+    TranslocoPipe,
+    TranslocoDirective,
+
+    WATT_DRAWER,
+    WattDatePipe,
+    WattProgressTrackerComponent,
+    WattDescriptionListComponent,
+    WattDescriptionListItemComponent,
+    WattProgressTrackerStepComponent,
+
+    VaterFlexComponent,
+
+    DhResultComponent,
+    DhProcessStateBadge,
+    DhEmDashFallbackPipe,
+    DhCalculationsDetailsGridAreasComponent,
+  ],
+  selector: 'dh-process-details',
+  styles: `
+    .headline {
+      display: inline-block;
+      padding-right: var(--watt-space-s);
+    }
+
+    dh-calculation-details-grid-areas {
+      margin: 0 var(--watt-space-ml);
+    }
+
+    watt-progress-tracker {
+      margin: 0 var(--watt-space-ml);
+    }
+  `,
+  template: `
+    @let process = result();
+    @let calculation = calculationDetails();
+    @let energyTimeSeriesRequest = energyTimeSeriesRequestDetails();
+    @let wholesaleRequest = wholesaleRequestDetails();
+
+    <watt-drawer *transloco="let t; read: 'devExamples.processes'" (closed)="closed()">
+      <watt-drawer-topbar>
+        @if (process) {
+          <dh-process-state-badge [status]="process.state">
+            {{ 'shared.states.' + process.state | transloco }}
```

Comment:
comment: from the perspective of someone inexperienced with the codebase, it could be confusing why we are using the transloco pipe here while also using the structural directive. I know we use it to "reach" global translations, but it always seemed a bit "quick-fixy" to me 😄 

also, I think `read` is deprecated and it is `prefix` now (at least if we are using the latest version).

Reply:
If you have a better solution I´m all ears :-) prefix doesnt work. Guess we are not one the latest version

Reply:
It's fine for now 🙂 I would like to move towards [translation scopes](https://jsverse.gitbook.io/transloco/advanced-topics/lazy-load/scope-configuration) and then just always using `t()`. That way we would save some initial loading and we could flatten the translation object by using multiple scopes instead. Something like this:

```html
<watt-drawer *transloco="let t;">
  <dh-process-state-badge [status]="process.state">
    {{ t('shared.states.' + process.state) }}

    ...

  <watt-description-list-item
    [label]="t('details.startedBy')"
```

In this setup, `shared` would be its own translation file, as could `details`.